handler/src/main/java/io/netty/handler/ssl/OpenSsl.java

```diff
@@ -149,8 +144,22 @@
                 long certBio = 0;
                 try {
                     try {
-                        SSLContext.setCipherSuite(sslCtx, TLSV13_CIPHERS, true);
-                        tlsv13Supported = true;
+                        StringBuilder tlsv13Ciphers = new StringBuilder();
+
+                        for (String cipher: TLSV13_CIPHERS) {
+                            String converted = CipherSuiteConverter.toOpenSsl(cipher, IS_BORINGSSL);
+                            if (converted != null) {
+                                tlsv13Ciphers.append(converted).append(':');
+                            }
+                        }
+                        if (tlsv13Ciphers.length() == 0) {
+                            tlsv13Supported = false;
+                        } else {
+                            tlsv13Ciphers.setLength(tlsv13Ciphers.length() - 1);
+                            SSLContext.setCipherSuite(sslCtx, tlsv13Ciphers.toString() , true);
+                            tlsv13Supported = true;
+                        }
+
                     } catch (Exception ignore) {
```

Comment:
Considering there is much more logic in the Try block, I don't know it's still okay to ignore this.  Maybe log?

Reply:
Let me just move most of it out of the block.